pkg/hostagent/mount.go

```diff
@@ -64,6 +65,10 @@ func (a *HostAgent) setupMount(m limayaml.Mount) (*mount, error) {
 		Readonly:            !(*m.Writable),
 		SSHFSAdditionalArgs: []string{"-o", sshfsOptions},
 	}
+	serverPath, err := exec.LookPath("sftp-server")
```

Comment:
This logic should be rather implemented in sshocker

Reply:
Also curious to know what distro needs this

Reply:
> Also curious to know what distro needs this

It is from Windows experiment, where I put all "userland" dependencies into "_output/bin/bundle" https://github.com/lima-vm/lima/issues/909#issuecomment-2616343798 I'm exploring how feasible is it to just provide all the dependencies with Lima bundled.

> This logic should be rather implemented in sshocker

I will create PR in sshocker then. Will keep this for now as a draft for references.

Reply:
PR in sshocker repo https://github.com/lima-vm/sshocker/pull/94